### DISCUSSION TITLE
Unsuccessful build on Win10 - aclocal-1.16 : error

### DISCUSSION THREAD

**Discussion by 17matejka** [AcademySoftwareFoundation/OpenRV#59] [Build Issues] [answered]
Hi,

I am new to coding and compiling, but I believe I am doing everything according to Win build tutorial, but still I cannot successfuly build the Open RV.
During my debug proces I find out, that for building PySide 2, program is using Python 3 installed in my OS, so I had to install pip requirments.txt to my instalation of Python 3 outside msys64 as well.
After this building looked good, installed Open RV folder has around 1GB, but still missing rv.exe etc.
I logged building into a file and find out this error message:

aclocal-1.16 : error : aclocal: file '/msys64/usr/share/aclocal/tcl.m4' does not exist [C:\OpenRV\cmake-build\cmake\dependencies\RV_DEPS_ATOMIC_OPS.vcxproj]
autoreconf-2.71 : error : aclocal failed with exit status: 1 [C:\OpenRV\cmake-build\cmake\dependencies\RV_DEPS_ATOMIC_OPS.vcxproj]
  -- Detecting CXX compiler ABI info
  Creating directories for 'RV_DEPS_DOCTEST'
  -- Looking for sys/types.h
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(241,5): error MSB8066: Custom build for 'C:\OpenRV\cmake-build\CMakeFiles\21d6eda148353c9194a
a30467b92b5cc\RV_DEPS_ATOMIC_OPS-mkdir.rule;C:\OpenRV\cmake-build\CMakeFiles\21d6eda148353c9194aa30467b92b5cc\RV_DEPS_ATOMIC_OPS-download.rule;C:\OpenRV\cmake-build\CMakeFiles\21d6eda148353c9194aa30467b92b5cc\
RV_DEPS_ATOMIC_OPS-update.rule;C:\OpenRV\cmake-build\CMakeFiles\21d6eda148353c9194aa30467b92b5cc\RV_DEPS_ATOMIC_OPS-patch.rule;C:\OpenRV\cmake-build\CMakeFiles\21d6eda148353c9194aa30467b92b5cc\RV_DEPS_ATOMIC_O
PS-configure.rule;C:\OpenRV\cmake-build\CMakeFiles\21d6eda148353c9194aa30467b92b5cc\RV_DEPS_ATOMIC_OPS-build.rule;C:\OpenRV\cmake-build\CMakeFiles\21d6eda148353c9194aa30467b92b5cc\RV_DEPS_ATOMIC_OPS-install.ru
le;C:\OpenRV\cmake-build\CMakeFiles\d6fd5551bcb9c62863b0231a2589ed33\RV_DEPS_ATOMIC_OPS-complete.rule;C:\OpenRV\cmake-build\CMakeFiles\192fb834f1f29087f091606513b4f96a\RV_DEPS_ATOMIC_OPS.rule;C:\OpenRV\cmake\d
ependencies\CMakeLists.txt' exited with code 1. [C:\OpenRV\cmake-build\cmake\dependencies\RV_DEPS_ATOMIC_OPS.vcxproj]

Any idea what am I doing wrong?

I attached log of rerun of the build, because full build log has 44MB.
[log.txt](https://github.com/AcademySoftwareFoundation/OpenRV/files/10708535/log.txt)

**Comment by paulbarton90** [comment, marked as answer]
For some reason people seem to be getting 2 paths in the ACLOCAL_PATH environment variable. It appears that you can just run `export ACLOCAL_PATH=/usr/share/aclocal` prior to the build command.  Originally from this comment - https://github.com/AcademySoftwareFoundation/OpenRV/discussions/67#discussioncomment-5014919.
